app/src/main/java/com/example/bookwormadventuresdeluxe2/MyBooksActivity.java

```diff
@@ -19,6 +20,8 @@
     RequestsFragment requestsFragment = new RequestsFragment();
     MyProfile profileFragment = new MyProfile();
 
+    public int LAUNCH_SCAN_ISBN = 1; // a code to check for result return
```

Comment:
This should not be declared twice, can you put this in its own file and have both files import it?